features/step_definitions/sign_in_steps.rb

```diff
@@ -0,0 +1,13 @@
+Given(/^I sign up for an account$/) do
+  visit new_user_registration_path
+  fill_in 'Name', with: 'Nubbe'
+  fill_in 'Email', with: 'nubbe@nubbe.com'
+  fill_in 'Password', with: '12345678'
+  fill_in 'Password confirmation', with: '12345678'
+  click_link_or_button 'Sign up'
+  click_link_or_button 'Sign Out'
```

Comment:
I would like you to explore 
* Nested Cucumber steps https://www.relishapp.com/cucumber/cucumber/docs/defining-steps/nested-steps
* Warden helpers (as we used in the Rails Messaging app) 
And choose one of these methods. I would also like you to make an argument why you prefer one over the other.